sound/soc/sof/ipc4-topology.c

```diff
@@ -1731,10 +1742,16 @@ sof_ipc4_prepare_copier_module(struct snd_sof_widget *swidget,
 				 * the tables in soc_acpi files depending on the _ADR and devID
 				 * registers for each codec.
 				 */
-				if (w->id == snd_soc_dapm_dai_in)
+				if (w->id == snd_soc_dapm_dai_in) {
```

Comment:
the comment above is no longer correct with the 0xc reference

Reply:
> the comment above is no longer correct with the 0xc reference

updated.

Reply:
Comment updated.